### PR TITLE
resolves STREAMS-401

### PR DESCRIPTION
**Pull request by steveblackmon**
incubator-streams-master updates to produce revamped streams website

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,20 +68,147 @@
     </prerequisites>
 
     <mailingLists>
-        <mailingList>
+      <!-- <mailingList>
```
Comment (eponvert) [inline pom.xml]: It's fine, but, is there a reason to keep the User Mailing List in there commented out?

File: pom.xml [hunk 2]
```diff
@@ -717,11 +1097,11 @@
                             </execution>
                         </executions>
                     </plugin>
-                    <plugin>
-                        <groupId>com.github.ferstl</groupId>
-                        <artifactId>depgraph-maven-plugin</artifactId>
-                        <version>${depgraph.plugin.version}</version>
-                    </plugin>
+                    <!--<plugin>-->
```
Comment (eponvert) [inline pom.xml]: same thing -- is it worth keeping this in here commented? if so, perhaps include a comment like "Uncomment this if you need to ...."

File: pom.xml (hunk 1 above)
Comment (steveblackmon) [inline pom.xml]: actually the appropriate action on this is to create a user mailing list!  I'll get that process rolling.

File: pom.xml (hunk 2 above)
Comment (steveblackmon) [inline pom.xml]: no reason to keep these commented out, will revert

File: pom.xml (hunk 1 above)
Comment (steveblackmon) [inline pom.xml]: Per http://www.apache.org/dev/committers.html#new-mailing-list the community needs to vote to create a new mailing list.  
I'll remove the commented out section for now until a user mailing list exists.

**Pull request merged**